crates/locutus-node/src/http_gateway.rs

```diff
@@ -206,19 +242,11 @@ struct NewSubscription {
 
 async fn process_client_request(
     client_id: ClientId,
-    msg: Result<Message, warp::Error>,
+    msg: Result<Message, axum::Error>,
     request_sender: &mpsc::Sender<ClientConnection>,
 ) -> Result<Option<Message>, Option<DynError>> {
     let msg = match msg {
-        Ok(m) if m.is_binary() || m.is_text() => m.into_bytes(),
-        Ok(m) if m.is_close() => return Err(None),
-        Ok(m) if m.is_ping() => {
-            return Ok(Some(Message::pong(vec![0, 3, 2])));
-        }
-        Ok(m) => {
-            tracing::debug!(msg = ?m, "received random message");
-            return Ok(None);
-        }
+        Ok(m) => m.into_data(),
```

Comment:
we don't need to hndle pings manually anymore?

Reply:
Fixed, it was incomplete